src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java

```diff
@@ -58,11 +61,12 @@ public class KNNQueryBuilder extends AbstractQueryBuilder<KNNQueryBuilder> {
     private final String fieldName;
     private final float[] vector;
     private int k = 0;
+    private float radius = 0.0f;
```

Comment:
can we call this variable as distance and other places where radius is used?

Reply:
@jmazanec15 For Lucene, the api requires score as parameter, so after distance -> score transformation finished, the value would be score instead of distance, I think it will also makes name confusing if all places called distance.